inst/rmarkdown/templates/isba/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+title: Short Paper for Bayesian Analyis
+author:
+  - firstname: Alice 
+    lastname: Anonymous
+    email: alice@example.com
+    url: www.example.com/aliceanon
+    affiliationref: addr1
+    footnoterefs: 
+      - t1
+      - t2
+      - m1
+  - firstname: Bob
+    lastname: Security
+    email: bob@example.com
+    affiliationref: addr1
+    footnoterefs: 
+      - t3
+      - m1
+      - m2
+  - firstname: Cat
+    lastname: Memes
+    email: cat@example.com
+    affiliationref: addr2
+    footnoterefs: 
+      - t1
+      - m2
+affiliations:
+  - ref: addr1
+    name: Some Institute of Technology 
+    address: Department, Street, City, State, Zip
+  - ref: addr2
+    name: Another University
+    address: Department, Street, City, State, Zip
+footnotes:
+  - ref: t1
+    text: "Corresponding Author"
+  - ref: t2
+    text: "Equal contribution"
+  - ref: t3
+    text: "Something"
+  - ref: m1
+    text: "Something"  
+  - ref: m2
+    text: "Something"
+MSC2020primary:
+  - 62C10
+  - 62F15
+MSC2020secondary:
+  - 60K35
+  - 91A27
+keywords:
+  - bayesian
+  - statistics
+supplements:
+  - title: Title of Supplement A
+    description: Short description of Supplement A.
+  - title: Title of Supplement B
+    description: Short description of Supplement B.
+doi: "10.31219/osf.io/wh3wr"
+arxiv: 2010.010101
+acknowledgements: |
+  And this is an acknowledgements section with a heading that was produced by the `acks` environment. Thank you all for helping me writing this Rmarkdown sample file.
+
+  Also thankful to God, mom and my cats.
+abstract: |
+  The abstract should summarize the contents of the paper. It should be clear, descriptive, self-explanatory and not longer than 200 words. It should also be suitable for publication in abstracting services. Please avoid using math formulas as much as possible.
+
+  This is a sample input file.  Comparing it with the output it generates can show you how to produce a simple document of your. That was the second paragraph.
+date: "`r Sys.Date()`"
+bibliography: sample.bib
+#linenumbers: true
+#numbersections: true
+csl: imsart.csl
```

Comment:
This is normally used with Pandoc's citeproc only. I don't think it has any effect here. Also `imsart.csl` is not provided in the package it seems, so it would error with something like file not found. It needs to be a url or provided.

Reply:
`imsart.csl` is in the skeleton directory.

Reply:
Yes, I overlooked the extension (need glasses). Yes, CSL is missing. I will remove it from the skeleton.